src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -991,6 +989,22 @@ private function getKernelRootHash(ContainerBuilder $container)
     }
 
     /**
+     * Gets the path of a given bundle using Bundle#getPath() when available.
+     *
+     * @param string $fqcn The bundle FQCN
+     *
+     * @return string
+     */
+    private function getBundlePath($fqcn)
+    {
+        $reflection = new \ReflectionClass($fqcn);
+        $getPathReflection = new \ReflectionMethod($fqcn, 'getPath');
+        $reflectionInstance = $reflection->newInstanceWithoutConstructor();
```

Comment:
ATM the only working way I found is to pass the Kernel instance as argument to the Framework bundle when register it (e.g. `new FrameworkBundle($this)` in the `AppKernel`), then pass it to a compiler pass and use it to get the bundle instances. It works but I think that adding an argument to the FrameworkBundle is really not an alternative :/